src/PaymentHandler/AbstractSynchronousPayonePaymentHandler.php

```diff
@@ -141,4 +142,17 @@ protected function handleResponse(
 
         $this->transactionDataHandler->saveTransactionData($paymentTransaction, $salesChannelContext->getContext(), $data);
     }
+
+    protected function createPaymentException(string $orderTransactionId, string $errorMessage, ?\Throwable $e = null): \Throwable
+    {
+        if (class_exists(PaymentException::class)) {
+            return PaymentException::asyncProcessInterrupted($orderTransactionId, $errorMessage, $e);
```

Comment:
Must be `syncProcessInterrupted` (not async...)

Reply:
missed that. thank you for keeping eyes on that.